content/docs/studio/user-guide/views/explore-experiments.md

```diff
@@ -0,0 +1,116 @@
+presented for you.
+
+![](https://static.iterative.ai/img/studio/view_components.png)
+
+## Components of a view
+
+The tabular display has different components that show you the complete
+experiment history as well as enable to you to generate plots, compare
+experiments, run new experiments, etc. The major components of a view are:
+
+- [Git history, which represents your experimentation history](#git-history)
+- [Display preferences to show/hide branches, commits and columns, and re-arrange your view](#display-preferences)
+- [Buttons to visualize, compare and run experiments](#visualize-compare-and-run-experiments)
+
+### Git history
+
+The branches and commits in your Git repository are displayed along with the
+corresponding metrics, files and hyperparameters.
+
+![](https://static.iterative.ai/img/studio/view_components_1.gif)
+
+### Display preferences
+
+The table contains buttons to specify filters and other preferences regarding
+which commits and columns to display.
+
+![](https://static.iterative.ai/img/studio/view_components_2.gif)
+
+#### **Filters:**
+
+You can filter the commits that you want to display by the following fields:
+
+- **Branch:** The Git branch
+- **Tag:** The Git tag
+- **Author:** Author of the Git commit
+- **Metric:** Values of different metrics. For instance, you can display only
+  those experiments for which the value of `avg_prec` is greater than `0.9`.
+- **Metric delta:** Change in the value of the metric. For instance, you can use
+  this filter to only display those experiments for which the value of
+  `avg_prec` changed by more than `0.1` compared to the baseline experiment.
+- **Param:** Values of different parameters
+- **File size:** Size of the data, model and other files corresponding to your
+  experiments
+- **File changed:** Whether or not any given file changed in the experiment
+
+#### **Columns:**
+
+Select the columns you want to display and hide the rest.
+![Showing and hiding columns](https://static.iterative.ai/img/studio/show_hide_columns.gif)
+
+Additionally, you can click and drag the columns in the table to rearrange them
+as per your preferences.
+
+If your view is missing some required columns or includes columns that you do
+not want, refer to the following troubleshooting sections to understand why this
+may have happened.
+
+- [View does not contain the columns that I want](/doc/studio/troubleshooting#view-does-not-contain-the-columns-that-i-want)
+- [View contains columns that I did not import](/doc/studio/troubleshooting#view-contains-columns-that-i-did-not-import)
+
+#### **Hide commits:**
+
+The following functionality are available for you to hide irrelevant commits
+from your views in DVC Studio.
+
+- **Hide commits and branches manually:** You can hide irrelevant commits, and
+  even entire branches. This means that commits you made simply to re-organize
+  your project structure will not take up space in your view anymore. To hide a
```

Comment:
> This means that commits you made simply to re-organize your project...

Quite complicated sentence, not easy to read and comprehend. Can this motivation be simplified and merged with the previous sentence?

p.s. avoid "simply" as rule of thumb.

Reply:
Changed